src/preprocessing/numerical.rs

```diff
@@ -0,0 +1,406 @@
+//! # Standard-Scaling For [RealNumber](../../math/num/trait.RealNumber.html) Matricies
+//! Transform a data [Matrix](../../linalg/trait.BaseMatrix.html) by removing the mean and scaling to unit variance.
+//!
+//! ### Usage Example
+//! ```
```

Comment:
Add the content of `fit_transform_with_default_standard_scaler` here using an example matrix

Reply:
yet another good catch I totally forgot to add the example in the docstring, as I originally planned!

Reply:
I added an example, I did not use `fit_transform_with_default_standard_scaler` because that is just a small function I wrote for convenience in my testing suite, I did not plan to make it public to other user.